caluma-demo/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:8
+
+COPY package.json yarn.lock /app/
+
+WORKDIR /app
+
+RUN yarn
+
+COPY . /app
+
+RUN yarn ember deploy production
+
+FROM nginx:alpine as final
```

Comment:
A second `FROM` in the Dockerfile seems wrong. Where is this coming from? What is the intention?

Reply:
Ok, as discussed.